admin/class-admin-apple-meta-boxes.php

```diff
@@ -188,22 +200,25 @@ public function publish_meta_box( $post ) {
 		?>
 		<div id="apple-news-publish">
 		<?php wp_nonce_field( $this->publish_action, 'apple_news_nonce' ); ?>
-		<?php
-			$section = new Apple_Actions\Index\Section( $this->settings );
-			try {
-				$sections = $section->get_sections();
-			} catch ( Apple_Actions\Action_Exception $e ) {
-				Admin_Apple_Notice::error( $e->getMessage() );
-			}
-
-			if ( ! empty( $sections ) ) :
-				?>
-				<h3><?php esc_html_e( 'Sections', 'apple-news' ) ?></h3>
-				<?php
-				self::build_sections_field( $sections, $post->ID );
-			endif;
-		?>
-		<p class="description"><?php esc_html_e( 'Select the sections in which to publish this article. Uncheck them all for a standalone article.' , 'apple-news' ) ?></p>
+		<?php self::build_sections_override( $post->ID ); ?>
+		<div class="apple-news-sections">
+			<?php
+				$section = new Apple_Actions\Index\Section( $this->settings );
+				try {
+					$sections = $section->get_sections();
+				} catch ( Apple_Actions\Action_Exception $e ) {
+					Admin_Apple_Notice::error( $e->getMessage() );
+				}
+
+				if ( ! empty( $sections ) ) :
+					?>
+					<h3><?php esc_html_e( 'Sections', 'apple-news' ) ?></h3>
+					<?php
+					self::build_sections_field( $sections, $post->ID );
```

Comment:
Also if the post doesn't match any mappings, it should default to Main as well. There should never be no sections selected or the post will go into oblivion.